llvm/lib/CodeGen/AtomicExpandPass.cpp

```diff
@@ -1764,6 +1769,9 @@ bool AtomicExpand::expandAtomicOpToLibcall(
   // The different signatures are built up depending on the
   // 'UseSizedLibcall', 'CASExpected', 'ValueOperand', and 'HasResult'
   // variables.
+  //
+  // Note: For CHERI hybrid mode we use __cap_atomic_* to handle cases where
```

Comment:
I do wonder if we should be using a `_c` suffix like we do for memcpy etc.

Reply:
Yeah that's probably better.